app/frontend/actions.js

```diff
@@ -266,29 +281,28 @@ module.exports = ({setState, getState}) => {
 
   const reloadWalletInfo = async (state) => {
     loadingAction(state, 'Reloading wallet info...')
-
-    const balance = await wallet.getBalance()
-    const ownAddressesWithMeta = await wallet.getFilteredVisibleAddressesWithMeta()
-    const transactionHistory = await wallet.getHistory()
-    const conversionRates = getConversionRates(state)
-
-    // timeout setting loading state, so that loading shows even if everything was cached
-    setTimeout(() => setState({loading: false}), 500)
-    setState({
-      balance,
-      ownAddressesWithMeta,
-      transactionHistory,
-    })
     try {
+      const balance = await wallet.getBalance()
+      const ownAddressesWithMeta = await wallet.getFilteredVisibleAddressesWithMeta()
+      const transactionHistory = await wallet.getHistory()
+      const conversionRates = getConversionRates(state)
+
+      // timeout setting loading state, so that loading shows even if everything was cached
+      setTimeout(() => setState({loading: false}), 500)
       setState({
-        conversionRates: await conversionRates,
+        balance,
+        ownAddressesWithMeta,
+        transactionHistory,
       })
+      await fetchConversionRates(state, conversionRates)
```

Comment:
fetchConversionRates takes one parameter

Reply:
Ah thanks, I removed the `state` argument from two occurrences, forgot to remove it in the third. I'll fix it.